### PR TITLE
feat: add make offer CTA and confirmation modal to inquiry conversation

### PR DESCRIPTION
**Pull request by lilyfromseattle**
# [PURCHASE-2333]

As a user I should be able to tap make offer and be presented with a confirm artwork dialog. Tapping cancel at any place should close the modal. The action for tapping confirm will be implemented in another ticket. 

Screen Shot 2021-01-19 at 5 27 09 PM
Screen Shot 2021-01-19 at 5 27 25 PM
Screen Shot 2021-01-19 at 5 27 34 PM

[PURCHASE-2333]: https://artsyproduct.atlassian.net/browse/PURCHASE-2333

### CODE REVIEW

File: src/lib/Scenes/Inbox/ConversationNavigator.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import React from "react"
 import { ConversationQueryRenderer as Conversation } from "./Screens/Conversation"
 
 export const ConversationNavigator: React.FC<{ conversationID: string }> = ({ conversationID }) => {
-  const initialRoute = { component: Conversation, passProps: { conversationID } }
+  const initialRoute = { component: Conversation, passProps: { conversationID }, title: "" }
```
Comment (lilyfromseattle) [inline src/lib/Scenes/Inbox/ConversationNavigator.tsx:6]: This is a fix for an issue I was having that would sometimes cause my build to crash. Title is required in initialRoute.

File: Podfile.lock [hunk 2]
```diff
@@ -416,6 +416,8 @@ PODS:
     - Mapbox-iOS-SDK (~> 6.3)
     - React
     - React-Core
+  - react-native-navigator-ios (1.0.0):
```
Comment (zephraph) [inline Podfile.lock:419]: These changes aren't expected. You'll need to rebase on master and re-run pod install.  

I wasn't sure about this so I had to ask: https://artsy.slack.com/archives/C02BAQ5K7/p1611153278020500

Comment (lilyfromseattle) [reply Podfile.lock:419]: Hmmm could this be why I was getting that error about the missing title?

Comment (lilyfromseattle) [reply Podfile.lock:419]: Rebase + pod install doesn't seem to have done anything

Comment (lilyfromseattle) [reply Podfile.lock:419]: gonna just reset the file to master and push it

**Review by zephraph — APPROVED** [review]

**Pull request merged**